packages/react/src/components/Tooltip/__tests__/DefinitionTooltip-test.js

```diff
@@ -11,11 +11,31 @@ import React from 'react';
 import { DefinitionTooltip } from '../DefinitionTooltip';
 
 describe('DefinitionTooltip', () => {
-  it('should display onClick a definition provided via prop', async () => {
+  it('should display on click a definition provided via prop', async () => {
     const definition = 'Uniform Resource Locator';
     render(<DefinitionTooltip definition={definition}>URL</DefinitionTooltip>);
+    expect(screen.getByRole('button')).toHaveAttribute(
+      'aria-expanded',
+      'false'
+    );
     await userEvent.click(screen.getByText('URL'));
-    expect(screen.getByText(definition)).toBeVisible();
+    expect(screen.getByRole('button')).toHaveAttribute('aria-expanded', 'true');
+  });
+
+  it('should control tooltip display when using keyboard event', async () => {
```

Comment:
I believe this test is already being handled here: https://github.com/carbon-design-system/carbon/pull/17679/files#diff-97685f64e2290de562987881b78ac2d0c7c60611353b92ae00de186126cef815R52-R70

Reply:
Oops, yeah it is! I've removed mine.